frontend/src/components/dashboard/data/Demo.vue

```diff
@@ -196,7 +196,9 @@ const changeCompare = (val: string) => {
           },
           color: {
             domain: colorDomain,
-            legend: true
+            // Can update categoricalColors variable with a custom color range (ex. range: categoricalColors)
```

Comment:
Does `scheme` override the `domain` attribute? The colors from `colorDomain` come from the Figma mock.

Reply:
Ah yes they do, I hadn't realized that was coming from Figma. I will revert that.

Reply:
> Ah yes it does, I hadn't realized the `colorDomain` was coming from Figma. I will revert that.
Adding an update here - the `scheme` and `range` set the colorways, while the `domain` maps the colors you've chosen to their values.

Reply:
Added a picture of how the plotColors look above, it looks like they match the chart on the homepage but not the dashboard mocks. Would it be good to check with UXD about that?

Reply:
Let's check with UXD eventually, but not as a blocker to merging this PR. Isaac, feel free to reopen if there are any outstanding concerns. Thank you both!